osi_trafficsign.proto

```diff
@@ -458,7 +458,7 @@ message TrafficSign
                 //
                 TYPE_DANGER_MOVABLE_BRIDGE = 101;
```

Comment:
Shouldn't this be LIFTING_BRIDGE?

Reply:
Well, the sign depicts a lifting bridge and it's named lifting bridge.

Reply:
I am actually a litle bit on the side of "generic expression including all kinds of moving bridges". I would assume that the "moving" part is what is relevant for the function. In the end all different signs for "moving bridges" can be expressed with one type?